### PR TITLE
Set main body background to --studio-white

### PR DESCRIPTION
**Pull request by DustyReagan**
Related to https://github.com/Automattic/dotcom-forge/issues/4221

## Proposed Changes

* While unifying the header, we want to update the background to match the header, which means updating the main body background for Calypso to `#fff` aka `--studio-white`.

Though this PR is part of the initiative to unify the site headers, no headers are modified in this PR.

Before | After
--|--
Screenshot 2023-10-17 at 1 39 44 PM |  Screenshot 2023-10-17 at 1 39 34 PM

## Testing Instructions

View the following pages and see that the background matches the Figma: eqlepL7MHifK8Bgr1OWqLr-fi-9424_29818

* http://calypso.localhost:3000/settings/general/
* http://calypso.localhost:3000/marketing/tools/
* http://calypso.localhost:3000/people/team/
* http://calypso.localhost:3000/activity-log/
* http://calypso.localhost:3000/posts/
* http://calypso.localhost:3000/media/
* http://calypso.localhost:3000/comments
* http://calypso.localhost:3000/me

Please look at other pages and ensure the background looks okay.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by obenland** [comment]
How does this line up with Nav Unification expectations and alignment with wp-admin? PCYsg-v8L-p2

**Comment by DustyReagan** [comment]
> How does this line up with Nav Unification expectations and alignment with wp-admin? PCYsg-v8L-p2

Thanks for the link! This change doesn't appear to override any custom color settings. However, `wp-admin` looks consistently set with a light gray background, while Calypso has a mix of light gray and white. I started a thread in Slack here to discuss further. p1697569190265249-slack-CKZHG0QCR

**Comment by DustyReagan** [comment]
Though we'd like to unify the background color in Calypso, we need to be sure it matches wp-admin (https://github.com/Automattic/wp-calypso/pull/83114#issuecomment-1767064085). I feel like this is a bit of a rabbit hole from our focus on unifying headers, so I'm closing this PR for now.

File: client/assets/stylesheets/_main.scss [hunk 1]
```diff
@@ -46,7 +46,7 @@ body,
 }
 
 body {
-	background: var(--color-surface-backdrop);
```
Comment (lsl) [inline client/assets/stylesheets/_main.scss:49]: Can we change the value of `--color-surface-backdrop` instead?

**Comment by lsl** [comment]
> I feel like this is a bit of a rabbit hole from our focus on unifying headers, so I'm closing this PR for now.

I think this should be in scope (if time allows it) - we're already going through every page to swap them over to the new header, might as well align on the same color background while we're at it.

Options I see:
- Align all pages on wp-admin gray including calypso pages currently on studio-white
- Align all pages on studio-white including wp-admin pages
- Align only calypso pages on studio-white, leaving "classic view" pages as wp-admin gray

I don't particularly mind which option we opt for but I don't like the idea of leaving some calypso pages using studio white and the others using wp-admin gray.

@nuriapenya could we defer to you for a design/product decision on which option we want to go for? Should calypso pages and classic view wp-admin pages match background color or are we ok keeping classic pages on a gray background and moving the rest to the white background per figma?

----

There are a few of these around that should be removed:

```
body.is-section-marketplace.theme-default.color-scheme {
	--color-surface-backdrop: var(--studio-white);
}
```

And the last step would be to change the surface-backdrop default in the color schemes package:

https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss#L191-L192

**Comment by obenland** [comment]
Based on my experience with Nav Unification I'm worried about background color changes leading to unexpected rabbit holes that would have to be chased to no end. Let's limit this effort to page titles for now please, I think that's a big step forward already.

**Comment by nuriapenya** [comment]
> @nuriapenya could we defer to you for a design/product decision on which option we want to go for? 

It seems that changing backgrounds might be too much of a scope for now, so I agree with @obenland, let's just focus on the page titles, and from design we can come up with a proper plan (check all pages, test white background, etc.) and work it on a different PR/project.

Would that work? 🙏🏽

**Pull request closed without merging**